src/java/test/org/apache/zookeeper/server/ServerMetricsTest.java

```diff
@@ -0,0 +1,91 @@
+package org.apache.zookeeper.server;
```

Comment:
Please add Apache License Header to this file. Missing of the header causes release audit warnings like [this](https://builds.apache.org/job/PreCommit-ZOOKEEPER-github-pr-build/2181/artifact/patchprocess/patchReleaseAuditProblems.txt)